contrib_bots/lib/joker.py

```diff
@@ -0,0 +1,52 @@
+# pip install schedule is necessary
+import schedule
+import time
+import json
+import threading
+import os
+from random import choice
+
+cache = os.path.expanduser('lib/Joker/jokes.json')
+
+def job(client):
+    with open(cache) as data_file:
+        joke = list(choice(json.load(data_file)).values())[0]
```

Comment:
hm... looks strange.
`json.load(data_file)` -> returns a list
`choice(above_list)` -> returns an elemt

Isn't it enough?